### PR TITLE
Add printing OpenLiberty repo commit hash for pingPerf

### PR DESCRIPTION
**Pull request by LongyuZhang**
- Add printing OpenLiberty repo commit hash for pingPerf
- Related Issue: Runtimes/backlog/1124

### CODE REVIEW

**Comment by LongyuZhang** [comment]
Test Link: hyc_grinder 33079 (output `Using dockerfile from OpenLiberty/ci.docker repo branch instanton with commit hash 2284b748da4d7ce7946ac45c8c052d5fce57082a`)

**Review by llxia — APPROVED** [review]
LGTM

File: external/criu/pingPerf.sh [hunk 1]
```diff
@@ -81,7 +81,10 @@ prepare() {
     git clone https://github.com/OpenLiberty/ci.docker.git
     (
         cd ci.docker || exit
-        git checkout instanton
+        openLibertyBranch="instanton"
```
Comment (karianna) [inline external/criu/pingPerf.sh:84]: This seems like an odd branch...

Comment (LongyuZhang) [reply external/criu/pingPerf.sh:84]: Openliberty `InstantOn` feature (`CRIU`) is still experimental beta version, so it is only available in this branch.

Comment (smlambert) [reply external/criu/pingPerf.sh:84]: a.k.a. "InstantOn (tm)"

;)

**Review by karianna — APPROVED** [review]
lgtm

**Pull request merged**